webpack/ForemanInventoryUpload/ForemanInventoryHelpers.js

```diff
@@ -0,0 +1,9 @@
+import { INVENTORY_BASE_URL } from './ForemanInventoryConstants';
+
+export const inventoryUrl = path => `${INVENTORY_BASE_URL}${path}`;
+
+export const inventoryStateWrapper = innerState => ({
```

Comment:
Should we have this function here? It's related only to tests.

Reply:
should we move it to a folder named `testHelpers`?
or maybe we can benefit from it being in the general helpers (?)